examples/stable/test/property/ic_api/crypto_get_random_values/test/tests.ts

```diff
@@ -47,19 +34,23 @@ export function getTests(): Test {
             }
 
             // If we're running the short test, only test Uint8Array
-            const typedArrays = shouldRunShortTest()
-                ? [{ name: 'Uint8Array', bytesPerElement: 1 }]
-                : [
-                      { name: 'Int8Array', bytesPerElement: 1 },
-                      { name: 'Uint8Array', bytesPerElement: 1 },
-                      { name: 'Uint8ClampedArray', bytesPerElement: 1 },
-                      { name: 'Int16Array', bytesPerElement: 2 },
-                      { name: 'Uint16Array', bytesPerElement: 2 },
-                      { name: 'Int32Array', bytesPerElement: 4 },
-                      { name: 'Uint32Array', bytesPerElement: 4 },
-                      { name: 'BigInt64Array', bytesPerElement: 8 },
-                      { name: 'BigUint64Array', bytesPerElement: 8 }
-                  ];
+            let typedArrays;
```

Comment:
Please don't use mutations like this, use const and don't mutate, just like you had it before, but keep the explicit conditional check